components/Feed/index.js

```diff
@@ -62,88 +62,79 @@ const greetingSubscription = gql`
   }
 `
 
-class Feed extends Component {
-  componentDidMount() {
-    this.subscribe()
+const Feed = ({
+  meta,
+  data: {
+    error,
+    loading,
+    greeting,
+    documents: connection,
+    fetchMore,
+    subscribeToMore
   }
+}) => {
+  const mapNodes = node => node.entity
 
-  componentDidUpdate() {
-    this.subscribe()
-  }
-
-  componentWillUnmount() {
-    this.unsubscribe && this.unsubscribe()
-  }
-
-  subscribe() {
-    if (!this.unsubscribe && this.props.data.greeting) {
-      this.unsubscribe = this.props.data.subscribeToMore({
-        document: greetingSubscription,
-        updateQuery: (prev, { subscriptionData }) => {
-          if (!subscriptionData.data) {
-            return prev
-          }
-          const { greeting } = subscriptionData.data.greeting
-          if (greeting) {
-            return {
-              ...prev,
-              greeting: {
-                ...greeting
-              }
+  useEffect(() => {
+    if (!subscribeToMore) {
+      return
+    }
+    let unsubscribe = subscribeToMore({
+      document: greetingSubscription,
+      updateQuery: (prev, { subscriptionData }) => {
+        if (!subscriptionData.data) {
+          return prev
+        }
+        const { greeting } = subscriptionData.data.greeting
+        if (greeting) {
+          return {
+            ...prev,
+            greeting: {
+              ...greeting
             }
-          } else {
-            return prev
           }
+        } else {
+          return prev
         }
-      })
+      }
+    })
+    return () => {
+      unsubscribe()
     }
-  }
+  }, [subscribeToMore])
 
-  render() {
-    const {
-      meta,
-      data: { error, loading, greeting, documents: connection, fetchMore }
-    } = this.props
+  return (
+    <Frame hasOverviewNav={true} raw meta={meta}>
```

Comment:
Generally we use the short hand for bool flags:
```suggestion
    <Frame hasOverviewNav raw meta={meta}>
```